lib/fluent/plugin/out_forward.rb

```diff
@@ -178,7 +178,7 @@ def start
       # But it should be overwritten by ack_response_timeout to rollback chunks after timeout
       if @ack_response_timeout && @delayed_commit_timeout != @ack_response_timeout
```

Comment:
ok, because these are configured intentionally by users, and don't cause problems for longer configured values.